numberOfProvinces.js

```diff
@@ -0,0 +1,88 @@
+/*
+n cities
+
+a - b - c
+
+i: n x n matrix
+o: total number of provinces
+
+directly or indirectly
+
+Input: isConnected = [[1,1,0],[1,1,0],[0,0,1]]
+Output: 2
+
+   0 1 2
+0 [1,1,0],
+1 [1,1,0],
+2 [0,0,1]
+
+*/
+
+const { Queue } = require('./utils/queue');
+
+const CONNECTED = 1;
+
+const getNeighbors = (node, matrix) => {
+    const currentRow = matrix[node].length;
+    const neighbors = [];
+
+    for (let neighborNode = 0; neighborNode < currentRow; neighborNode++) {
```

Comment:
These are not necessarily neighbors, they're just _potential_ neighbors (we still need to check if we have a `1` for that potential neighbor). Make sure to name this something clear, for example `potentialNeighborNode`.